hedera-mirror-rest/model/index.js

```diff
@@ -16,6 +16,8 @@ import ContractStateChange from './contractStateChange';
 import ContractTransactionHash from './contractTransactionHash';
 import CryptoTransfer from './cryptoTransfer';
 import CustomFee from './customFee';
+import CustomFeeLimits from './customFeeLimits.js';
+import FixedFee from './fixedFee.js';
```

Comment:
nit, also order them alphabetically

```suggestion
import CustomFeeLimits from './customFeeLimits';
import FixedFee from './fixedFee';
```